ml-data-hub-plugin/src/main/groovy/com/marklogic/gradle/task/ClearUserModulesTask.groovy

```diff
@@ -0,0 +1,14 @@
+package com.marklogic.gradle.task
+
+import com.marklogic.hub.impl.DataHubImpl
+
+class ClearUserModulesTask extends AbstractConfirmableHubTask {
+
+    @Override
+    void executeIfConfirmed() {
+        println "Clearing user modules. This requires a user with at least 'data-hub-developer' role."
```

Comment:
I think technically, the user just needs the data-hub-module-writer role and the rest-writer privilege. But I think it's good to say data-hub-developer here because for modules, I don't think we want to advertise anything less than the data-hub-developer role.